doc/Installation/Installation-CentOS-7-Apache.md

```diff
@@ -10,7 +10,7 @@ path: blob/master/doc/
 
     rpm -Uvh https://mirror.webtatic.com/yum/el7/webtatic-release.rpm
 
-    yum install composer cronie fping git httpd ImageMagick jwhois mariadb mariadb-server mtr MySQL-python net-snmp net-snmp-utils nmap php72w php72w-cli php72w-common php72w-curl php72w-gd php72w-mbstring php72w-mysqlnd php72w-process php72w-snmp php72w-xml php72w-zip python-memcached rrdtool
+    yum install composer cronie fping git httpd ImageMagick jwhois mariadb mariadb-server mtr MySQL-python net-snmp net-snmp-utils nmap php72w php72w-cli php72w-common php72w-curl php72w-gd php72w-mbstring php72w-mysqlnd php72w-process php72w-snmp php72w-xml php72w-zip python-memcached rrdtool php72w-bcmath
```

Comment:
Why have you added bcmath?

Reply:
Because without it I got the following error : `production.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to undefined function PhpAmqpLib\Wire\bcmod() in /opt/librenms/vendor/php-amqplib/php-amqplib/PhpAmqpLib/Wire/AMQPWriter.php:36`

But as it seems to be AMQP specific maybe it's overkill to add it to basic install.

Reply:
Huh, it is used by amqp...  maybe not the default install docs but somewhere else.